ADOPTERS.md

```diff
@@ -5,8 +5,10 @@
 Below is a list of companies and projects that have adopted Open Cluster Management(OCM). If you have been using OCM and your name is not on this list, send us a PR!
 
 - [Alibaba Cloud](https://www.alibabacloud.com/)
+- [Andy Anderson](https://linkedin.com/in/clubanderson)
 - [Ant Group](https://www.antgroup.com/)
 - [AppsCode Inc.](https://appscode.com)
```

Comment:
```suggestion
- [AppsCode Inc.](https://appscode.com)
- [Kubestellar](https://docs.kubestellar.io)
```

Reply:
Sorry, I misunderstood, I think https://github.com/open-cluster-management-io/ocm/pull/501#issuecomment-2199542059 is right. It should be the Company/Project name and website.